taxonomybot.py

```
@@ -0,0 +1,656 @@
+#!/home/wikibot/pywikibot-env/bin/python3.9
+import requests
+import re
+import time
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)